I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/extremis/coteries/transhumanistes/evts_UnivTranshumanistes.rpy", line 1, in script
    init -5 python:
  File "game/extremis/coteries/transhumanistes/evts_UnivTranshumanistes.rpy", line 5, in <module>
    from extremis.religions import religion
ImportError: No module named religions

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/extremis/coteries/transhumanistes/evts_UnivTranshumanistes.rpy", line 1, in script
    init -5 python:
  File "D:\Dev\Renpy\renpy-7.3.5-sdk\renpy\ast.py", line 914, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\Dev\Renpy\renpy-7.3.5-sdk\renpy\python.py", line 2028, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/extremis/coteries/transhumanistes/evts_UnivTranshumanistes.rpy", line 5, in <module>
    from extremis.religions import religion
ImportError: No module named religions

Windows-8-6.2.9200
Ren'Py 7.3.5.606
 
Sun Jan 31 20:54:22 2021
